massgen/tests/test_position_bias_calibration.py

```
@@ -0,0 +1,149 @@
+        scores = {
+            "agent1": {"E1": _score(9), "E2": _score(5)},  # total 14
+            "agent2": {"E1": _score(7), "E2": _score(8)},  # total 15
+        }
+        best, _, _ = _extract_flat_scores(scores, item_prefix="E", n_items=2)
+        assert best == "agent2"
+
+
+class TestPresentationOrderCounterbalancing:
+    """Defect 2: presentation order must counterbalance with order_seed."""
+
+    summaries = {"agent_a": "AAA", "agent_b": "BBB", "agent_c": "CCC"}
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)